### DISCUSSION TITLE
Convert IPA symbols into audio sound

### DISCUSSION THREAD

**Discussion by pabloab** [pndurette/gTTS#433] [Ideas]
I came across [this popular question](https://linguistics.stackexchange.com/q/1549) about how to convert [IPA symbols](https://en.wikipedia.org/wiki/International_Phonetic_Alphabet) into audio sound. One of the tools that can do this is espeak/[-ng](https://github.com/espeak-ng/espeak-ng/blob/master/docs/phonemes.md), so wondering if gTTS can do it too seems a natural consequence.

**Comment by Zireael07** [comment]
Seconding
